Anna_Mayilyan/python/25_05/Pascal.py

```diff
@@ -0,0 +1,11 @@
+n = int(input("n= "))
+row = []
+for i in range(n):
+    row.append( [] )
+    row[i].append(1)
+    for j in range(1, i):
+        row[i].append(row[i - 1][j - 1] + row[i - 1][j])
+    if (n != 0):
+        row[i].append(1)
+
+print(row)
```

Comment:
- No validation for inputs
![image](https://user-images.githubusercontent.com/80816144/119726202-ea8b7a00-be81-11eb-8241-d53f0f458f40.png)

- The outupt isn't triangle
![image](https://user-images.githubusercontent.com/80816144/119726232-f37c4b80-be81-11eb-9101-2929bd994779.png)


- For big data the program is killed 
![image](https://user-images.githubusercontent.com/80816144/119726312-0e4ec000-be82-11eb-8bd5-c82749972e21.png)